db/018-refactor-sessions.sql

```diff
@@ -0,0 +1,67 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 18;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        ALTER TABLE sessions
+            DROP still_counter,
+            DROP od_flag,
```

Comment:
FYI here is a Slack thread where Gordon indicates the possible desire to continue to track old-style sessions: https://bravely.slack.com/archives/CAN15J43F/p1624662155122500?thread_ts=1624658926.121400&cid=CAN15J43F

I'm trying to convince him otherwise :smile: